### PR TITLE
[nl] add param to skip getting related things

### PR DESCRIPTION
**Pull request by chejennifer**

### CODE REVIEW

File: static/nodejs_server/query.ts [hunk 1]
```diff
@@ -294,6 +294,7 @@ export async function getQueryResult(
     idx,
     mode,
     varThreshold,
+    skipRelatedThings: wantRelatedQuestions ? "" : "True",
```
Comment (shifucun) [inline static/nodejs_server/query.ts:297]: nit: use "true"

File: static/nodejs_server/query.ts [hunk 2]
```diff
@@ -330,6 +331,7 @@ export async function getQueryResult(
       enclosedPlaceType;
   }
   const relatedThings = nlResp.data["relatedThings"] || {};
+  console.log(relatedThings);
```
Comment (shifucun) [inline static/nodejs_server/query.ts:334]: remove logging?

File: server/routes/explore/api.py [hunk 3]
```diff
@@ -118,6 +118,8 @@ def detect_and_fulfill():
       Params.EXP_MORE_DISABLED.value] = request.get_json().get(
           Params.EXP_MORE_DISABLED, "")
   utterance.insight_ctx[Params.DC.value] = dc_name
+  utterance.insight_ctx[Params.SKIP_RELATED_THINGS] = request.args.get(
```
Comment (shifucun) [inline server/routes/explore/api.py:121]: should here set the value to be a boolean while checking `lower(Params.SKIP_RELATED_THINGS.value) == 'true'`?

Comment (chejennifer) [reply server/routes/explore/api.py:121]: done

File: static/nodejs_server/query.ts (hunk 2 above)
Comment (chejennifer) [reply static/nodejs_server/query.ts]: done

File: static/nodejs_server/query.ts (hunk 1 above)
Comment (chejennifer) [reply static/nodejs_server/query.ts]: done

**Review by shifucun — APPROVED** [review]

File: server/routes/explore/api.py (hunk 3 above)
Comment (pradh) [reply server/routes/explore/api.py:121]: nit: use lower()?

**Pull request merged**